Failure looks real:
```
======================================================================

ERROR: test_sms_registration (corehq.apps.sms.tests.BackendTestCase)

----------------------------------------------------------------------

Traceback (most recent call last):

  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/sms/tests/__init__.py", line 451, in test_sms_registration

    incoming("+9991234567", "JOIN {} WORKER tester".format(self.domain), "TEST_CASE_BACKEND")

  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/sms/api.py", line 440, in incoming

    process_incoming(msg, delay=delay)

  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/sms/api.py", line 521, in process_incoming

    verify.process_verification(v, msg)

  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/sms/verify.py", line 74, in process_verification

    if v.verified:

AttributeError: 'NoneType' object has no attribute 'verified'
```

--- a/corehq/apps/sms/verify.py
+++ b/corehq/apps/sms/verify.py
@@ -71,7 +71,7 @@ def send_verification(domain, user, phone_number, logged_event):
 
 
 def process_verification(v, msg):
-    if v.verified:
+    if not v or v.verified:
         return
 
     logged_event = MessagingEvent.get_current_verification_event(

